default.nix

```
@@ -0,0 +1,83 @@
+let
+  inputs = import ./flake-inputs.nix;
+in
+  {
+    nixpkgs ? inputs.nixpkgs,
+    dream2nix ? inputs.dream2nix,
+    rust-overlay ? inputs.rust-overlay,
+    sops-nix ? inputs.sops-nix,
+    self ? {
+      dirtyRev = "dirty";
+      lastModifiedDate = "19700101";
+    },
+    system ? builtins.currentSystem,
+  }: let
+    lib = import (nixpkgs + "/lib");
```

If you have `pkgs` from arguments you get

```suggestion
    inherit (pkgs) lib;
```